misc/scripts/install-local.sh

```diff
@@ -55,6 +55,21 @@ function checks() {
 	if [[ -z "$hash" ]]; then
 		fancy_message warn "Package does not contain a hash"
 	fi
+	if [[ -z "$description" ]]; then
+		fancy_message warn "Package does not contain a description"
+	fi
+	if [[ -z "$version" ]]; then
+		fancy_message error "Package does not contain version"
+		exit 1
+	fi
+	if [[ -z "$name" ]]; then
+		fancy_message error "Package does not contain name"
+		exit 1
+	fi
+	if [[ -z "$url" ]]; then
+		fancy_message error "Package does not contain URL"
+		exit 1
+	fi
```

Comment:
```suggestion
	fi
	if [[ -z "$maintainer" ]]; then
		fancy_message warn "Package does not have a maintainer"
		fancy_message warn "It maybe no longer maintained. Please be advised."
	fi
```

I think we could add a warning for missing `maintainer` fields. So if some non-`-git` pacscript gets no longer updated for quite some time. We could remove that field, and the users will be warned accordingly.

Or the maintainer themselves could remove the field if they no longer wish to maintain the script.

Reply:
This could be one line

Reply:
What do you suggest for the one liner?

Reply:
Too much is cramped into a single line. I prefer 2 line approach.